astro/src/content/docs/apis/_application-request-body.mdx

```diff
@@ -227,11 +228,11 @@ import Xmlsignaturec14nmethodValues from 'src/content/docs/_shared/_xmlSignature
      * com.myApp://example
      * com.myApp:/example
 
-    <span class="text-green-600">Available since 1.32.0</span>
+    <Aside type="version">Available since 1.32.0</Aside>
```

Comment:
I like that the `Aside` stands out more, but I think it is less clear that it's referring to the paragraph below.

Would it make sense to rework the `Aside` a bit for this use case? Instead of what we've got today
<img width="256" alt="image" src="https://github.com/user-attachments/assets/a6b70eb7-4a08-461d-bd3b-ba02c4ff7fbb" />

we could replace the title of **version** with the version number or maybe **Since 1.32.0**, and then put the paragraph inside the `Aside` body.

Reply:
took advice, created a second PR: https://github.com/FusionAuth/fusionauth-site/pull/3665